### PR TITLE
feat: reduce access token size and set null to empty string

### PR DESCRIPTION
**Pull request by gtn1024**
See: https://github.com/casdoor/casdoor/pull/1172

### CODE REVIEW

File: src/main/java/org/casbin/casdoor/entity/CasdoorUser.java [hunk 1]
```diff
@@ -64,18 +64,18 @@ public class CasdoorUser implements Serializable {
...
-    private String wechat;
-    private String facebook;
-    private String dingtalk;
-    private String weibo;
-    private String gitee;
-    private String linkedin;
-    private String wecom;
-    private String lark;
-    private String gitlab;
+    private String github = "";
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/entity/CasdoorUser.java:67]: Can you sync all the fields from Go?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/main/java/org/casbin/casdoor/entity/CasdoorUser.java (hunk 1 above)
Comment (gtn1024) [reply src/main/java/org/casbin/casdoor/entity/CasdoorUser.java:67]: done

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@gtn1024 demo video?

**Comment by gtn1024** [comment]
@hsluoyz I used [casdoor-spring-boot-example](https://github.com/casdoor/casdoor-spring-boot-example) with [casdoor-spring-boot-starter](https://github.com/casdoor/casdoor-spring-boot-starter) to test the code, and make some changes in `pom.xml`, seems great.

image

https://user-images.githubusercontent.com/69113276/193435614-8fbd59e2-54af-4aa9-a128-83864abde0d4.mp4

**Pull request merged**